### PR TITLE
Prakt03 tasks4-6 Stefka Lyaskalieva

### PR DESCRIPTION
**Pull request by stefka04**
Stefka Lyaskalieva
4MI0600158 Group 2

### CODE REVIEW

File: SI/Sem.03/Pract.03/Prakt3Task05.cpp [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	int newNumber;
+	cout << "Enter a number between [0,100]: "<<endl;
+	cin >> number;
+	cout << "Enter a number: "<<endl;
+	cin >> newNumber;
+	if (number >= 0 && number <= 100)
+	{
+		while (number != newNumber)
+		{ 
+			cout << "The number of attempts is: " << attempts << endl;
```
Comment (plamena-ilieva) [inline SI/Sem.03/Pract.03/Prakt3Task05.cpp:16]: You do not need to print the attempts each time, only after the correct number is entered.

File: SI/Sem.03/Pract.03/Prakt3Task05.cpp [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+			cout<<(number > newNumber ? "The new number is smaller." : "The number is bigger") << endl;
+		    cin >> newNumber;
+		 
+		}
+		cout << "The number of attempts is: "<<attempts;
+	}
+	else
+	{
+		cout << "The number is not in the range!  ";
+		return 0;
```
Comment (GeorgiTerziev02) [inline SI/Sem.03/Pract.03/Prakt3Task05.cpp:27]: Useless return 0

File: SI/Sem.03/Pract.03/Prakt3Task05.cpp [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	cin >> number;
+	cout << "Enter a number: "<<endl;
+	cin >> newNumber;
+	if (number >= 0 && number <= 100)
+	{
+		while (number != newNumber)
+		{ 
+			cout << "The number of attempts is: " << attempts << endl;
+			attempts++;
+			cout<<(number > newNumber ? "The new number is smaller." : "The number is bigger") << endl;
```
Comment (GeorgiTerziev02) [inline SI/Sem.03/Pract.03/Prakt3Task05.cpp:18]: You have an extra print "The number is bigger" when you enter the equal number

File: SI/Sem.03/Pract.03/Prakt3Task06.cpp [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+{
+	int n, newn, newSum, sum;
+	sum = 0;
+	newSum = 0;
+	cout << "Enter a number: " << endl;
+	cin >> n;
+	for (int i = 1; i <= n - 1; i++)
+	{
+		cin >> newn; 
+		newSum += newn;
```
Comment (GeorgiTerziev02) [inline SI/Sem.03/Pract.03/Prakt3Task06.cpp:13]: This solution can be optimized to use only one sum variable

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]

**Comment by GeorgiTerziev02** [comment]
The correct solutions have already been uploaded, check them in the link below
https://github.com/Justsvetoslavov/Introduction_to_programming/tree/master/SI/Sem.03/Pract.03

**Pull request closed without merging**